wire-schema/src/jvmMain/java/com/squareup/wire/schema/EnumType.kt

```diff
@@ -49,12 +51,21 @@ class EnumType private constructor(
     allowAlias = options.get(ALLOW_ALIAS)
   }
 
-  override fun validate(linker: Linker) {
+  override fun validate(linker: Linker, syntax: Syntax?) {
     val linker = linker.withContext(this)
 
     if ("true" != allowAlias) {
       validateTagUniqueness(linker)
     }
+    if (syntax == PROTO_3) {
+      validateTagZero(linker)
+    }
+  }
+
+  private fun validateTagZero(linker: Linker) {
```

Comment:
`validateTagZeroExistence` is better? 🙄

Reply:
Let's check that the first constant is a zero as well.

Reply:
`validateFirstElement()` might be enough?